packages/paste-website/src/pages/patterns/navigation/index.mdx

```diff
@@ -0,0 +1,276 @@
+
+## Usage
+
+### General
+
+The Navigation pattern structures common elements found in Twilio products to provide customers a cohesive, expected way to find what they’re looking for. Applying this pattern across all our experiences also makes the Twilio product suite feel and behave like one family of products.
+
+### What’s a product, experience, or feature?
+
+- **Products** or **product platforms**: Twilio Console, Flex, Segment, and SendGrid.
+- **Experiences**: All the products above, as well as Twilio Docs and internal tools.
+- **Product features**: Something with a distinct Twilio name (e.g., Voice, Engage) or groups of task-based features (e.g., Queue stats, Insights)
+- **Product-wide features**: Controls that apply to the entire product, like search, support, and user settings.
+
+These are different from Twilio’s internal definition of these terms, but are used in these guidelines for simplicity.
+
+### What’s an account, workspace, or user?
+
+At Twilio, we define accounts, users, and workspaces as:
+
+- **Account**: In Twilio Console, a group of resources that are administered and billed together. Similar to workspaces in Segment. [Read more about accounts.](https://support.twilio.com/hc/en-us/articles/360011132374-Getting-Started-with-Twilio-Accounts-and-Subaccounts)
+- **Workspace**: In Segment, a group of sources that are administered and billed together. Similar to accounts in Twilio Console. [Read more about workspaces.](https://segment.com/docs/guides/#whats-a-workspace)
+- **User**: A person with a login. Users can have access to multiple accounts or workspaces.
+
+### Layout and composition
+
+To decide where to place new navigation elements, use these general principles:
+
+- The **Sidebar** is where customers navigate between product features using [Sidebar Navigation](/components/sidebar-navigation), or where they control Sidebar visibility.
+- The **left side of the Topbar** is where customers navigate to account or workspace controls.
+- The **right side of the Topbar** is where customers find product- or experience-wide features, like search, [Product Switcher](/components/product-switcher), and [User Dialog](/components/user-dialog).
+- **Navigating within a product feature** should be handled at the page-level, with components like [In-Page Navigation](/components/in-page-navigation) and [Breadcrumb](/components/breadcrumb).
+
+<ResponsiveImage src={NavigationWireframe} aria-hidden="true" alt="" />
+
+### Accessibility
+
+The Sidebar and Topbar navigation components have the following considerations:
+
+- Ensure the [Skip links](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) are [correctly implemented](/components/sidebar#skip-links). This ensures users can easily and quickly access the main content of a web page by bypassing repeated blocks of content such as navigation links.
+- Ensure the `SidebarNavigation` component is labelled with an accessible name.
+
```

Comment:
suggestion: Add a `## Ingredients` here because the Accessibility section looks like it's just kinda shoved in between everything right now.